Dockerfile

```diff
@@ -3,6 +3,7 @@ COPY . /usr/src/app
 VOLUME /usr/src/app
 EXPOSE 4567
 
+ENV BUNDLER_VERSION=2.1.4
```

Comment:
should this be committed?

Reply:
I got a docker-error regarding saying "you have to user bundler version 2+". But since I've got another problem now it did not fix the main issue. So I removed it. Thank you!